templates/cisco_asa_show_vpn-sessiondb.textfsm

```diff
@@ -0,0 +1,55 @@
+Value ANYCONNECT_CLIENT_INACTIVE (\d+)
+Value SSL_TLS_DTLS_ACTIVE (\d+)
+Value SSL_TLS_DTLS_CUMULATIVE (\d+)
+Value SSL_TLS_DTLS_PEAK_CONCURRENT (\d+)
+Value SSL_TLS_DTLS_PEAK_INACTIVE (\d+)
+Value CLIENTLESS_VPN_ACTIVE (\d+)
+Value CLIENTLESS_VPN_CUMULATIVE (\d+)
+Value CLIENTLESS_VPN_PEAK_CONCURRENT (\d+)
+Value VPN_BROWSWER_ACTIVE (\d+)
+Value VPN_BROWSWER_CUMULATIVE (\d+)
+Value VPN_BROWSWER_PEAK_CONCURRENT (\d+)
+Value TOTAL_ACTIVE_AND_INACTIVE (\d+)
+Value TOTAL_CUMULATIVE (\d+)
+Value DEVICE_TOTAL_VPN_CAPACITY (\d+)
+Value DEVICE_LOAD_PERCENT (\d+)
+Value TUNNELS_SUMMARY_CLIENTLESS_ACTIVE (\d+)
+Value TUNNELS_SUMMARY_CLIENTLESS_CUMULATIVE (\d+)
+Value TUNNELS_SUMMARY_CLIENTLESS_PEAK_CONCURRENT (\d+)
+Value TUNNELS_SUMMARY_ANYCONNECT_PARENT_ACTIVE (\d+)
+Value TUNNELS_SUMMARY_ANYCONNECT_PARENT_CUMULATIVE (\d+)
+Value TUNNELS_SUMMARY_ANYCONNECT_PARENT_PEAK_CONC (\d+)
+Value TUNNELS_SUMMARY_SSLTUNNEL_ACTIVE (\d+)
+Value TUNNELS_SUMMARY_SSLTUNNEL_CUMULATIVE (\d+)
+Value TUNNELS_SUMMARY_SSLTUNNEL_PEAK_CONCURRENT (\d+)
+Value TUNNELS_SUMMARY_DTLSTUNNEL_ACTIVE (\d+)
+Value TUNNELS_SUMMARY_DTLSTUNNEL_CUMULATIVE (\d+)
+Value TUNNELS_SUMMARY_DTLSTUNNEL_PEAK_CONCURRENT (\d+)
+Value TOTALS_ACTIVE (\d+)
+Value TOTALS_CUMULATIVE (\d+)
+
+Start
+  ^\s*AnyConnect\s+Client\s+:\s+${ANYCONNECT_CLIENT_ACTIVE}\s+:\s+${ANYCONNECT_CLIENT_CUMULATIVE}\s+:\s+${ANYCONNECT_CLIENT_PEAK_CONCURRENT}\s+:\s+${ANYCONNECT_CLIENT_INACTIVE}\s*$$
+  ^\s*SSL/TLS/DTLS\s+:\s+${SSL_TLS_DTLS_ACTIVE}\s+:\s+${SSL_TLS_DTLS_CUMULATIVE}\s+:\s+${SSL_TLS_DTLS_PEAK_CONCURRENT}\s+:\s+${SSL_TLS_DTLS_PEAK_INACTIVE}\s*$$
+  ^\s*Clientless\s+VPN\s+:\s+${CLIENTLESS_VPN_ACTIVE}\s+:\s+${CLIENTLESS_VPN_CUMULATIVE}\s+:\s+${CLIENTLESS_VPN_PEAK_CONCURRENT}\s*$$
+  ^\s*Browser\s+:\s+${VPN_BROWSWER_ACTIVE}\s+:\s+${VPN_BROWSWER_CUMULATIVE}\s+:\s+${VPN_BROWSWER_PEAK_CONCURRENT}\s*$$
+  ^\s*Total\s+Active\s+and\s+Inactive\s+:\s+${TOTAL_ACTIVE_AND_INACTIVE}\s+Total\s+Cumulative\s+:\s+${TOTAL_CUMULATIVE}\s*$$
+  ^\s*Device\s+Total\s+VPN\s+Capacity\s+:\s+${DEVICE_TOTAL_VPN_CAPACITY}
+  ^\s*Device\s+Load\s+:\s+${DEVICE_LOAD_PERCENT}%\s*$$
+  ^\s*Clientless\s+:\s+${TUNNELS_SUMMARY_CLIENTLESS_ACTIVE}\s+:\s+${TUNNELS_SUMMARY_CLIENTLESS_CUMULATIVE}\s+:\s+${TUNNELS_SUMMARY_CLIENTLESS_PEAK_CONCURRENT}
+  ^\s*AnyConnect\-Parent\s+:\s+${TUNNELS_SUMMARY_ANYCONNECT_PARENT_ACTIVE}\s+:\s+${TUNNELS_SUMMARY_ANYCONNECT_PARENT_CUMULATIVE}\s+:\s+${TUNNELS_SUMMARY_ANYCONNECT_PARENT_PEAK_CONC}\s*$$
+  ^\s*SSL\-Tunnel\s+:\s+${TUNNELS_SUMMARY_SSLTUNNEL_ACTIVE}\s+:\s+${TUNNELS_SUMMARY_SSLTUNNEL_CUMULATIVE}\s+:\s+${TUNNELS_SUMMARY_SSLTUNNEL_PEAK_CONCURRENT}\s*$$
+  ^\s*DTLS\-Tunnel\s+:\s+${TUNNELS_SUMMARY_DTLSTUNNEL_ACTIVE}\s+:\s+${TUNNELS_SUMMARY_DTLSTUNNEL_CUMULATIVE}\s+:\s+${TUNNELS_SUMMARY_DTLSTUNNEL_PEAK_CONCURRENT}\s*$$
+  ^\s*Totals\s+:\s+${TOTALS_ACTIVE}\s+:\s+${TOTALS_CUMULATIVE}\s*$$ -> Record
+  ^\s*$$
+  # Lines that are not capturing output but in the output
+  ^\s*\-{75}\s*$$ -> NoRecord
```

Comment:
You shouldn't have to have both L49 and L50. You don't need to account for the number of occurrences of the dash just that more than one may exist and that will cover all the instances.
```suggestion
  ^\s*\-+ -> NoRecord
```